src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -184,6 +184,8 @@ gui.appliedenergistics2.Lumen=Lumen
 gui.appliedenergistics2.Empty=Empty
 gui.appliedenergistics2.Stored=Stored
 gui.appliedenergistics2.Cancel=Cancel
+gui.appliedenergistics2.ETA=ETA:
+gui.appliedenergistics2.ETAFormat=HH:mm:ss
```

Comment:
No, the duration format needs to be in ISOwhatevernumber format otherwise it will fail.
We could change it to `ETA: %s` so the order could be changed. But the only other option for format would be actually not make it available.